### PR TITLE
feat(ONYX-658): update notification list item design

### PR DESCRIPTION
**Pull request by dariakoko**
This PR resolves [ONYX-658] 

### Description

Update notification list item design
| iOS | Android |
| --- | --- |
| image | IMAGE 2024-01-25 16:56:21 |

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [ ] I added an **[app state migration]**, or my changes do not require one.
- [ ] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [ ] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- update notification list item design (behind a feature flag) - daria

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[ONYX-658]: https://artsyproduct.atlassian.net/browse/ONYX-658?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (update notification list item design (behind a feature flag) - daria)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 162e5a061302f9a0aec6d28a456ab1a42ef8cd19
</p>

File: data/schema.graphql [hunk 1]
```diff
@@ -3983,13 +3983,15 @@ type CollectorProfileType {
   email: String
   emailConfirmed: Boolean
     @deprecated(reason: "emailConfirmed is going to be removed, use isEmailConfirmed instead")
+  firstNameLastInitial: String
```
Comment (olerichter00) [inline data/schema.graphql:3986]: 👀 

The schema update should happen automatically and not be part of this PR. Mabye a `gith rebase main` could help 🤔

**Review by olerichter00 — DISMISSED** [review]
Looks great 🌟 

I've only added a comment about the schema update.

File: data/schema.graphql (hunk 1 above)
Comment (dariakoko) [reply data/schema.graphql:3986]: Yeah, it didn't happen for me until I updated it with a command
Will try again 👌

Comment (dariakoko) [reply data/schema.graphql]: dooone

File: src/app/Scenes/Activity/utils/getNotificationTypeLabel.ts [hunk 2]
```diff
@@ -13,6 +13,34 @@ export const getNotificationTypeLabel = (notificationType: NotificationTypesEnum
...
+export const getNotificationTypeBadge = (notificationType: NotificationTypesEnum) => {
+  switch (notificationType) {
+    case "PARTNER_OFFER_CREATED":
+      return "Limited Time Offer"
+    default:
+      return null
+  }
+}
+
+export const getNewNotificationTypeLabel = (notificationType: NotificationTypesEnum) => {
```
Comment (nickskalkin) [inline src/app/Scenes/Activity/utils/getNotificationTypeLabel.ts:25]: **suggestion**: should we move it to metaphysics to avoid having such logic in both Force/Eigen?

**Review by olerichter00 — APPROVED** [review]

File: src/app/Scenes/Activity/ActivityItem.tsx [hunk 3]
```diff
@@ -49,6 +55,83 @@ export const ActivityItem: React.FC<ActivityItemProps> = (props) => {
...
+                      backgroundColor="blue10"
+                      alignSelf="flex-start"
+                    >
+                      <Text variant="xs" color="blue100" px={0.5}>
+                        {getNotificationTypeBadge(item.notificationType)}
+                      </Text>
+                    </Flex>
+                  )}
+
+                  <Text variant="sm-display" fontWeight="bold">
```
Comment (nickskalkin) [inline src/app/Scenes/Activity/ActivityItem.tsx:105]: **minor suggestion**: Maybe it's just me, but images and headline text seem to be too close to each other. I would add a 10px top margin.

| Spec | Your design |
|---|---|
|  Screenshot 2024-01-26 at 10 27 23 | Screenshot 2024-01-26 at 10 27 34 |

Comment (nickskalkin) [reply src/app/Scenes/Activity/ActivityItem.tsx:105]: and maybe **bold** is not necessary 🤔

**Review by nickskalkin — COMMENTED** [review]
All good! Left two comments

File: src/app/Scenes/Activity/ActivityItem.tsx (hunk 3 above)
Comment (dariakoko) [reply src/app/Scenes/Activity/ActivityItem.tsx:105]: You're right, thanks for noticing
Fixed
image
